client/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Name-Request",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bcgov/namerequest.git"
+  },
+  "version": "0.1.0",
+  "private": true,
+  "license": "Apache License, Version 2.0",
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
+    "test:e2e": "vue-cli-service test:e2e",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "babel-plugin-transform-decorators": "^6.24.1",
+    "core-js": "^3.3.2",
+    "material-design-icons-iconfont": "^5.0.1",
+    "roboto-fontface": "*",
+    "vue": "^2.6.10",
+    "vue-class-component": "^7.0.2",
+    "vue-fragment": "^1.5.1",
+    "vue-property-decorator": "^8.3.0",
+    "vue-router": "^3.1.3",
+    "vuetify": "^2.1.0",
+    "vuex": "^3.0.1",
+    "vuex-class": "^0.3.2",
+    "vuex-module-decorators": "^0.11.0"
+  },
+  "devDependencies": {
+    "@types/jest": "^24.0.11",
+    "@vue/cli-plugin-babel": "^4.0.0",
+    "@vue/cli-plugin-e2e-nightwatch": "^4.0.0",
+    "@vue/cli-plugin-eslint": "^4.0.0",
+    "@vue/cli-plugin-router": "^4.0.0",
+    "@vue/cli-plugin-typescript": "^4.0.0",
+    "@vue/cli-plugin-unit-jest": "^4.0.0",
+    "@vue/cli-plugin-vuex": "^4.0.0",
+    "@vue/cli-service": "^4.0.0",
+    "@vue/eslint-config-typescript": "^4.0.0",
+    "@vue/test-utils": "1.0.0-beta.29",
```

Comment:
You may also consider this package to help your unit testing: https://github.com/AmpleOrganics/vue-test-utils-helpers.git